projects/control-service/projects/versions-of-external-dependencies.gradle

```diff
@@ -31,7 +31,7 @@ project.ext {
             'com.cronutils:cron-utils'                                           : 'com.cronutils:cron-utils:9.2.1',
             'net.lingala.zip4j:zip4j'                                            : 'net.lingala.zip4j:zip4j:2.11.5',
             'net.javacrumbs.shedlock:shedlock-provider-jdbc-template'            : 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:5.11.0',
-            'net.javacrumbs.shedlock:shedlock-spring'                            : 'net.javacrumbs.shedlock:shedlock-spring:5.11.0',
+            'net.javacrumbs.shedlock:shedlock-spring'                            : 'net.javacrumbs.shedlock:shedlock-spring:5.10.0',
```

Comment:
Do you needd to rebase the branch? How did this show up. 

When updating the branch please use the rebase option (not the merge - in the Github UI the merge is the default one you need to click to select rebase)

Reply:
I might have added it mistakenly, fixed now